draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -167,6 +167,12 @@ For any remote ICE candidate received by the ICE agent, the following procedure
 
 4. Otherwise, ignore the candidate.
 
+An ICE agent may add additional restrictions to resolve the ICE candidate using
+mDNS as this mechanism allows attackers to send network traffic to devices with
+well-know mDNS names. If the source providing the ICE candidates is not trusted,
```

Comment:
Won't all sources be untrusted to some extent, e.g., the remote peer?

Reply:
Also related to the mentioning of DNS-SD, how about the possibility of doing something similar to STUN authentication, with out-of-band credential? Given that we are already doing it for STUN, it is not impossible to include a TXT record in the mDNS message for authentication, which does not require any spec change of mDNS itself.

Reply:
The source providing the candidates is not really the remote peer, it is the webrtc provider that will send the candidates from one endpoint to another.
Various trust rules can be applied. For instance, an HTTPS page with non mixed content might be considered trusted, an HTTP page might not be considered trusted.